### DISCUSSION TITLE
新的xhttp协议如何，可以取代xtls-rprx-vision吗

### DISCUSSION THREAD

**Discussion by CarbonPool** [XTLS/Xray-core#4207] [General]
一直在用xtls-rprx-vision很久了，看到xhttp写的介绍很多，安全性，速度，延迟，这方面会比xtls-rprx-vision优秀吗

**Comment by kuraudowelle** [comment]
目前还不行，或者是目前给出的配置方案还不成熟。
试了HK US SG三个地区的机器，首先在同一条宽带（移动）同一时间段测试，无论是延迟还是速度，XHTTP都要逊色于Vision。
很纳闷于是又在另一条宽带（电信）同等条件和前提下测试，结果还是一样。
特别是传输速度，似乎受到地理位置的影响很大，HK的XHTTP降幅最小，SG次之，US最大（最多会下降80%以上）

**Reply by kuraudowelle** [reply]
从第三版文章正式出来，连续测试了3天，百思不得其解。
PS：传输速度降幅指的是同一个机器XHTTP与Vision比较下的降幅。下载的测试文件和链接都是一样的，大小1GB，千兆宽带环境。

**Reply by iKira** [reply]
贴出你的配置看看

**Reply by kuraudowelle** [reply]
就是照抄五合一里面Benjamin的那个vless+xhttp配置，还有lxhao的也照抄过了，其他都没动过

**Reply by RPRX** [reply]
v24.12.18 的 XHTTP 在多路复用上有一些 bug，main 分支已修复，这个月还有一版

**Comment by janesle** [comment]
没人回复啊，有一个Q群讨论：939989248
